src/components/header/Header.jsx

```diff
@@ -8,8 +9,26 @@ export default class Header extends Component {
         <div className="navbar-brand">
           <img src={logo} alt="Logo" className="navbar-item image is-rounded is-64x64" />
           <h1 className="navbar-item">Musaeum</h1>
-        </div>
+          </div>
+          <div className="navbar-menu">
+            <div className="navbar-start">
+              <a className="navbar-item">Museos</a>
+              <a className="navbar-item">Artefacto</a>
+              <a className="navbar-item">Monumentos</a>
+            </div>
+            <div className="navbar-end" id="navbarEndLogIn">
```

Comment:
Maybe some other bulma component inside?

Reply:
It's is progress, soon will be completed the navbar.